tests/Unit/Evolution/Systems/Cce/Test_LinearSolve.cpp

```diff
@@ -319,9 +339,27 @@ void test_pole_integration_with_linear_operator(
                                           Spectral::Quadrature::GaussLobatto>(
                  number_of_radial_grid_points));
 
-  const auto random_angular_data = make_with_random_values<ComplexDataVector>(
+  auto random_angular_data = make_with_random_values<ComplexDataVector>(
```

Comment:
I found this new code block a little unclear, and thought it could benefit from a comment. I think it could be helpful to say (in a sentence or two) why you are generating random data to go into a filter buffer, and also what the significance is of truncating the random modes below.